sc2/sc2process.py

```diff
@@ -34,17 +34,33 @@ def add(cls, value):
 
     @classmethod
     def kill_all(cls):
-        logger.info("kill_switch: Process cleanup")
+        logger.info(f"kill_switch: Process cleanup for {len(cls._to_kill)} processes")
         for p in cls._to_kill:
             p._clean()
 
 
 class SC2Process:
+    """
+    A class for handling SCII applications.
+
+    :param host: hostname for the url the SCII application will listen to
+    :param port: the websocket port the SCII application will listen to
+    :param fullscreen: whether to launch the SCII application in fullscreen or not, defaults to False
+    :param resolution: (window width, window height) in pixels, defaults to (1024, 768)
+    :param placement: (x, y) the distances of the SCII app's top left corner from the top left corner of the screen
+                       e.g. (20, 30) is 20 to the right of the screen's left border, and 30 below the top border
+    :param render:
+    :param sc2_version:
+    :param base_build:
+    :param data_hash:
+    """
     def __init__(
         self,
         host: str = "127.0.0.1",
         port: Optional[int] = None,
         fullscreen: bool = False,
+        resolution: Optional[Iterable[int]] = None,
+        placement: Optional[Iterable[int]] = None,
```

Comment:
I think with the type hint `iterable` all of `set, tuple, list, dict, generators` are meant, but only on `list` and `tuple` indexing with integers works.
I would change it to `Optional[Union[List[int], Tuple[int, int]]]`